mongo/integration/mtest/mongotest.go

```diff
@@ -84,6 +84,7 @@ type T struct {
 	maxServerVersion string
 	validTopologies  []TopologyKind
 	auth             *bool
+	enterprise       *bool
```

Comment:
I would be fine assuming enterprise for running driver tests.

Reply:
I'd like to leave this here because we don't usually run enterprise versions locally.